public/scripts/client.js

```diff
@@ -17,4 +19,8 @@ $(() => {
   loadTasks("incomplete");
 
   loadTasksByCategory();
+
+  // $("#header-text").click(()=> {
+  //   makeTMDBRequest();
+  // });
```

Comment:
Is this just something pending that still needs work? Maybe add a comment on the line above to make a note.
```suggestion
  // TODO: Things needed for this to work, or why it's commented out
  // $("#header-text").click(()=> {
  //   makeTMDBRequest();
  // });
```

Reply:
oh yeah this was for testing purposes, forgot to delete